README.md

```diff
@@ -37,4 +39,9 @@ with an `(I)` appended e.g. `EMIS (I)`. The addition of `(I)` represents a GP
 that is now using the Informatica system.
 
 Note: The list above was created by running
-`jq -c '[.[].Supplier] | unique ' output/pomi.json`
+`jq -c '[.[].Supplier] | unique ' html/json/pomi.json`
+
+The json files created will be available via the container's webserver at
```

Comment:
It's missing a description along with Booking and scripts at lines 13-14. I'll add the missing line